### PR TITLE
Bump Figures version 0.3.15 and README

### PR DESCRIPTION
**Pull request by johnbaldwin**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -8,6 +8,15 @@ Reporting and data retrieval app for `Open edX <https://open.edx.org/>`__.
 
 .. _notice_section:
 
+11 Sep 2020 - Figures release 0.3.15
+====================================
+
+* Learner progress overview UI improvements, add to main menu
+
+  * https://github.com/appsembler/figures/pull/255
+  * https://github.com/appsembler/figures/pull/256
```
Comment (grozdanowski) [inline README.rst:17]: @johnbaldwin this PR is not to be merged at this point. It is to be merged upon actual release.

**Review by grozdanowski — CHANGES_REQUESTED** [review]
PR 256 should not be merged in yet.

File: README.rst (hunk 1 above)
Comment (johnbaldwin) [reply README.rst:17]: @grozdanowski Ok, so the scope of Figures 0.3.15 will be just PR #255?

**Review by OmarIthawi — APPROVED** [review]

**Comment by johnbaldwin** [comment]
@grozdanowski Would you please approve this PR if you are satisfied with the README correction so I don't have to do an administrator override to commit?

**Review by thraxil — APPROVED** [review]

**Review by grozdanowski — APPROVED** [review]

**Pull request merged**